tests/apps/actorapp/go.mod

```diff
@@ -0,0 +1,7 @@
+module app
```

Comment:
Do we need this go.mod for every test app in each app folder .. ?

Reply:
yes